js/dragManager.js

```diff
@@ -0,0 +1,169 @@
+/**
+ * Created by Ruslan on 29-Mar-16.
+ */
+  
```

Comment:
The DragManager is not working correctly yet if there two open cards in one row and you want tot split them and take only one